<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>-1</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Video Call</string>
       </property>
       <property name="icon">
        <iconset resource="buddylist.qrc">
         <normaloff>:/chatWindow/resources/camera-web.png</normaloff>:/chatWindow/resources/camera-web.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Audio Call</string>
       </property>
       <property name="icon">
        <iconset resource="buddylist.qrc">
         <normaloff>:/chatWindow/resources/audio-headset.png</normaloff>:/chatWindow/resources/audio-headset.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Photos</string>
       </property>
       <property name="icon">
        <iconset resource="buddylist.qrc">
         <normaloff>:/chatWindow/resources/preferences-desktop-personal.png</normaloff>:/chatWindow/resources/preferences-desktop-personal.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="contactStatus">
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="buddylist.qrc">:/status/resources/user-offline.png</pixmap>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="contactName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="contactStatusText">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>148</red>
             <green>148</green>
             <blue>148</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>148</red>
             <green>148</green>
             <blue>148</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>169</red>
             <green>167</green>
             <blue>167</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="handleWidth">
        <number>3</number>
       </property>
       <property name="childrenCollapsible">
        <bool>false</bool>
       </property>
       <widget class="QWebView" name="messagesView">
        <property name="url">
         <url>
          <string>about:blank</string>
         </url>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QToolButton" name="toolButton_4">
             <property name="toolTip">
              <string>Emoticons</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="buddylist.qrc">
               <normaloff>:/chatWindow/resources/face-smile.png</normaloff>:/chatWindow/resources/face-smile.png</iconset>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_5">
             <property name="toolTip">
              <string>IMVironments</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="buddylist.qrc">
               <normaloff>:/chatWindow/resources/view-presentation.png</normaloff>:/chatWindow/resources/view-presentation.png</iconset>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_6">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="buddylist.qrc">
               <normaloff>:/chatWindow/resources/preferences-desktop-font.png</normaloff>:/chatWindow/resources/preferences-desktop-font.png</iconset>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_7">
             <property name="toolTip">
              <string>BUZZ!!</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="buddylist.qrc">
               <normaloff>:/chatWindow/resources/appointment-reminder.png</normaloff>:/chatWindow/resources/appointment-reminder.png</iconset>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_8">
             <property name="toolTip">
              <string>Send file...</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="buddylist.qrc">
               <normaloff>:/chatWindow/resources/mail-attachment.png</normaloff>:/chatWindow/resources/mail-attachment.png</iconset>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_9">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="buddylist.qrc">
               <normaloff>:/chatWindow/resources/ksnapshot.png</normaloff>:/chatWindow/resources/ksnapshot.png</iconset>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_10">
             <property name="toolTip">
              <string>Hide avatars</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="buddylist.qrc">
               <normaloff>:/chatWindow/resources/arrow-right.png</normaloff>:/chatWindow/resources/arrow-right.png</iconset>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="textEditBkgnd" native="true">
              <property name="styleSheet">
               <string notr="true">QWidget#textEditBkgnd {
background-color: palette(Base);
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QTextEdit" name="textEdit">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>125</height>
                  </size>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="horizontalScrollBarPolicy">
                  <enum>Qt::ScrollBarAlwaysOff</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="sendButton">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>50</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Send</string>
                 </property>
                 <property name="autoDefault">
                  <bool>false</bool>
                 </property>
                 <property name="default">
                  <bool>false</bool>
                 </property>
                 <property name="flat">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="hisAvatar">
         <property name="maximumSize">
          <size>
           <width>96</width>
           <height>96</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="myAvatar">
         <property name="maximumSize">
          <size>
           <width>96</width>
           <height>96</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>textEdit</tabstop>
  <tabstop>sendButton</tabstop>
  <tabstop>toolButton_4</tabstop>
  <tabstop>toolButton_5</tabstop>
  <tabstop>toolButton_6</tabstop>
  <tabstop>toolButton_7</tabstop>
  <tabstop>toolButton_8</tabstop>
  <tabstop>toolButton_9</tabstop>
  <tabstop>toolButton_10</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>toolButton_2</tabstop>
  <tabstop>toolButton_3</tabstop>
  <tabstop>messagesView</tabstop>
 </tabstops>
 <resources>
  <include location="buddylist.qrc"/>
 </resources>
 <connections/>
</ui>
